docs/getting-started/agentless-troubleshooting.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[agentless-integration-troubleshooting]]
+= Agentless integrations FAQ
+
+Frequently asked questions and troubleshooting steps for {elastic-sec}'s agentless CSPM integration.
+
+*When I make a new integration, how long until the agent appears on the Integration Policies page?*
```

Comment:
Should these be H2 titles? That'd allow for the questions to display in the mini-nav on the right side and would make navigating the page a bit easier.